client/state/action-types.js

```diff
@@ -462,55 +463,55 @@ export const JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS =
 export const JETPACK_CREDENTIALS_UPDATE_FAILURE = 'JETPACK_CREDENTIALS_UPDATE_FAILURE';
-export const JETPACK_CREDENTIALS_GET = 'JETPACK_CREDENTIALS_GET';
-export const JETPACK_CREDENTIALS_GET_SUCCESS = 'JETPACK_CREDENTIALS_GET_SUCCESS';
-export const JETPACK_CREDENTIALS_GET_FAILURE = 'JETPACK_CREDENTIALS_GET_FAILURE';
+export const JETPACK_CREDENTIALS_UPDATE_SUCCESS = 'JETPACK_CREDENTIALS_UPDATE_SUCCESS';
 export const JETPACK_DISCONNECT_RECEIVE = 'JETPACK_DISCONNECT_RECEIVE';
 export const JETPACK_DISCONNECT_REQUEST = 'JETPACK_DISCONNECT_REQUEST';
 export const JETPACK_DISCONNECT_REQUEST_FAILURE = 'JETPACK_DISCONNECT_REQUEST_FAILURE';
 export const JETPACK_DISCONNECT_REQUEST_SUCCESS = 'JETPACK_DISCONNECT_REQUEST_SUCCESS';
-export const JETPACK_MODULES_RECEIVE = 'JETPACK_MODULES_RECEIVE';
-export const JETPACK_MODULES_REQUEST = 'JETPACK_MODULES_REQUEST';
-export const JETPACK_MODULES_REQUEST_FAILURE = 'JETPACK_MODULES_REQUEST_FAILURE';
-export const JETPACK_MODULES_REQUEST_SUCCESS = 'JETPACK_MODULES_REQUEST_SUCCESS';
 export const JETPACK_MODULE_ACTIVATE = 'JETPACK_MODULE_ACTIVATE';
 export const JETPACK_MODULE_ACTIVATE_FAILURE = 'JETPACK_MODULE_ACTIVATE_FAILURE';
 export const JETPACK_MODULE_ACTIVATE_SUCCESS = 'JETPACK_MODULE_ACTIVATE_SUCCESS';
 export const JETPACK_MODULE_DEACTIVATE = 'JETPACK_MODULE_DEACTIVATE';
 export const JETPACK_MODULE_DEACTIVATE_FAILURE = 'JETPACK_MODULE_DEACTIVATE_FAILURE';
 export const JETPACK_MODULE_DEACTIVATE_SUCCESS = 'JETPACK_MODULE_DEACTIVATE_SUCCESS';
+export const JETPACK_MODULES_RECEIVE = 'JETPACK_MODULES_RECEIVE';
+export const JETPACK_MODULES_REQUEST = 'JETPACK_MODULES_REQUEST';
+export const JETPACK_MODULES_REQUEST_FAILURE = 'JETPACK_MODULES_REQUEST_FAILURE';
+export const JETPACK_MODULES_REQUEST_SUCCESS = 'JETPACK_MODULES_REQUEST_SUCCESS';
 export const JETPACK_PRODUCT_INSTALL_REQUEST = 'JETPACK_PRODUCT_INSTALL_REQUEST';
 export const JETPACK_PRODUCT_INSTALL_STATUS_RECEIVE = 'JETPACK_PRODUCT_INSTALL_STATUS_RECEIVE';
 export const JETPACK_PRODUCT_INSTALL_STATUS_REQUEST = 'JETPACK_PRODUCT_INSTALL_STATUS_REQUEST';
 export const JETPACK_REMOTE_INSTALL = 'JETPACK_REMOTE_INSTALL';
 export const JETPACK_REMOTE_INSTALL_FAILURE = 'JETPACK_REMOTE_INSTALL_FAILURE';
 export const JETPACK_REMOTE_INSTALL_SUCCESS = 'JETPACK_REMOTE_INSTALL_SUCCESS';
 export const JETPACK_SCAN_ENQUEUE_REQUEST = 'JETPACK_SCAN_ENQUEUE_REQUEST';
-export const JETPACK_SCAN_ENQUEUE_REQUEST_SUCCESS = 'JETPACK_SCAN_ENQUEUE_REQUEST_SUCCESS';
 export const JETPACK_SCAN_ENQUEUE_REQUEST_FAILURE = 'JETPACK_SCAN_ENQUEUE_REQUEST_FAILURE';
+export const JETPACK_SCAN_ENQUEUE_REQUEST_SUCCESS = 'JETPACK_SCAN_ENQUEUE_REQUEST_SUCCESS';
 export const JETPACK_SCAN_ENQUEUE_UPDATE = 'JETPACK_SCAN_ENQUEUE_UPDATE';
 export const JETPACK_SCAN_HISTORY_REQUEST = 'JETPACK_SCAN_HISTORY_REQUEST';
-export const JETPACK_SCAN_HISTORY_REQUEST_SUCCESS = '	JETPACK_SCAN_HISTORY_REQUEST_SUCCESS';
```

Comment:
cc @Automattic/jetpack-voyager - we believe this leading whitespace in the action type value was unintentional, so we're removing it. Feel free to revert that our assumption is incorrect 😉